src/storage/admin/StatisTask.cpp

```diff
@@ -170,18 +186,27 @@ StatisTask::genSubTask(GraphSpaceID spaceId,
             auto ranking = NebulaKeyUtils::getRank(vIdLen, key);
             auto destination = NebulaKeyUtils::getDstId(vIdLen, key).str();
 
-            if (source == lastSrcVertexId &&
-                edgeType == lastEdgeType &&
-                ranking == lastRank &&
-                destination == lastDstVertexId) {
-                // Multi version
-            } else {
+            if (source != lastSrcVertexId ||
+                edgeType != lastEdgeType ||
+                ranking != lastRank ||
+                destination != lastDstVertexId) {
                 spaceEdges++;
                 edgetypeEdges[edgeType] += 1;
                 lastSrcVertexId = source;
                 lastEdgeType  = edgeType;
                 lastRank = ranking;
                 lastDstVertexId = destination;
+
+                uint64_t vid = 0;
+                if (isIntId) {
+                    memcpy(static_cast<void*>(&vid), destination.data(), 8);
+                } else {
+                    nebula::MurmurHash2 hash;
+                    vid = hash(destination.data());
```

Comment:
It is best to use `toString `here,  refer https://github.com/vesoft-inc/nebula-storage/pull/248

Reply:
line 187, I has convertted